lib/websql/WebSQLDatabase.js

```diff
@@ -33,9 +33,19 @@ function WebSQLDatabase(dbVersion, db) {
 WebSQLDatabase.prototype._onTransactionComplete = function(err) {
   var self = this;
 
-  function done() {
-    if (err) {
-      self._currentTask.errorCallback(err);
+  function done(_err, results) {
```

Comment:
Why is this `_err` unchecked? Is there some situation where it could be truthy?

Reply:
Ah I see, the rollback or the commit themselves could return an error presumably. I think this is worth handling, but I can do that in the other PR.

Reply:
Basically this boiled down to finding a reproducible path which I wasn’t able to do. I also tested this against an Android/iOS client but `err` didn’t contain anything at least in my particular case.